src/InputProcessor.java

```diff
@@ -0,0 +1,84 @@
+/*
+* https://docs.oracle.com/javase/tutorial/essential/environment/cmdLineArgs.html
+* https://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java
+* Bold text https://stackoverflow.com/a/29109958 - http://ascii-table.com/ansi-escape-sequences.php
+*/
+class InputProcessor {
+        void validateInput(String[] args) {
+                if (args.length > 0 ) {
+
+                        // We could compare here when args[0] be help
+                        //We compare if args[0] match with insert command
+                        if (args[0].equals(DataBasicCommands.insert.name())){   // https://stackoverflow.com/a/9858135
+                                // We check then 2nd argument exists
+                                if( args.length == 2) {
+
+                                        if ( !args[1].isEmpty() ){
+
+                                                System.out.println("Path to JSON file args[1] = " + args[1]);
+
+                                                // Then we check the validity of the path, this can be in a method
+
+                                        } else {
+                                                System.out.println("Error: \u001B[1mYou have to specify a correct path\u001B[0m");
```

Comment:
I'm guessing the `\u001B[1m` bit changes the colour?
Pro tip: Try using `System.err` instead of `System.out` for errors.

Reply:
Okay, so I know now that this isn't for errors.
How about making a `makeBold` method so that you can put the weird `\u001B[1m` stuff in only one place?

```
String makeBold(String s) {
    //Implementation here
}
```

Reply:
Hi!, yes this is a really good idea. I will do it.

Reply:
Done!